app/controllers/notifications_controller.rb

```diff
@@ -40,17 +40,18 @@ def destroy
     redirect_to notifications_url
   end
 
-  def publish
+  def release
     @notification = Notification.find(params[:id])
-    @notification.publish!
```

Comment:
@hengsokly please kindly using this style of code block and wrap it in the begin/rescue and providing message whether success or failure